specification/common-types/resource-management/v5/mobo.json

```diff
@@ -0,0 +1,38 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "5.0",
+    "title": "Common types"
+  },
+  "paths": {},
+  "definitions": {
+    "ManagedOnBehalfOfConfiguration": {
+      "type": "object",
+      "description": "Managed-On-Behalf-Of configuration properties. This configuration exists for the resources where a resource provider manages those resources on behalf of the resource owner.",
+      "properties": {
+        "moboBrokerResources": {
```

Comment:
Since its inside the context of a `ManagedOnBehalfOfConfiguration` type already, can we just shorten the name to 'brokerResources' or 'brokers'?

Reply:
I think it makes more sense to have it with "Resources" as other configuration inside it may not refer directly resources. So, having it as resources makes it more indicative of a resource.